.gitignore

```diff
@@ -205,3 +205,22 @@ Chart.lock
 
 # Might be generated when you build wheels
 pip-wheel-metadata
+
+# tmp ...
+Backups/
+Cache/
+CachedData/
+CachedExtensions/
+Code Cache/
+Cookies
+Cookies-journal
+GPUCache/
+Network Persistent State
+Session Storage/
+TransportSecurity
+User/
+languagepacks.json
+machineid
+rapid_render.json
+storage.json
+tmp.md
```

Comment:
@SalAlba can you please remove this change? It's unrelated to the PR :)

Reply:
This is done